pkg/services/ngalert/store/alert_rule.go

```diff
@@ -261,6 +263,17 @@ func (st DBstore) GetOrgAlertRules(ctx context.Context, query *ngmodels.ListAler
 	})
 }
 
+func (st DBstore) GetRuleGroups(ctx context.Context, query *ngmodels.ListRuleGroupsQuery) error {
+	return st.SQLStore.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		ruleGroups := make([]string, 0)
+		if err := sess.SQL(`SELECT DISTINCT rule_group FROM alert_rule`).Find(&ruleGroups); err != nil {
```

Comment:
Can you please rewrite in xorm?
```suggestion
		if err := sess.Table("alert_rule").Distinct("alert_group").Find(&ruleGroups); err != nil {
```